tsdate/demography.py

```diff
@@ -111,6 +120,17 @@ def __init__(self, population_size, time_breaks=None):
         self.coalescent_breaks = coalescent_breaks
         self.coalescent_rate = coalescent_rate
 
+    def as_init_params(self):
```

Comment:
Maybe `as_dict` for this method name?

Reply:
True. I guess this entirely contains all the info we need, so `as_dict` is correct?